### PR TITLE
cronhook, privacy chores

### PR DESCRIPTION
**Pull request by nils-wisiol**
Added cronhook that executs the privacy chores.

Chosen 2:55am UTC without any particular reason.

Environment get dumped into container at /etc/environment. Potential security thread? It contains sensitive data.

Closes nils-wisiol/desec-internal#9.

### CODE REVIEW

File: api/cronhook/start-cron.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# start-cron.sh
+
+printenv >> /etc/environment
```
Comment (peterthomassen) [inline api/cronhook/start-cron.sh]: As you already said in https://github.com/desec-io/desec-stack/pull/38#issue-220883537, this writes out sensitive data. Can you justify why it is needed?

Comment (nils-wisiol) [inline api/cronhook/start-cron.sh]: Django needs to talk to `dbapi` to perform correctly, and it needs credentials to do so. Those credentials are stored in the environment; however cron does not by default provide the env it is running in to the started cron jobs. Hence, we need to pass on the env to the command line django by ourselves.

I will also argue that it does not pose an additional security thread: an attacker that can read from `/etc/environment` can likely also read env variables.

**Comment by nils-wisiol** [comment]
rebased onto updated #37

**Comment by peterthomassen** [comment]
lgtm

**Pull request merged**